applications/FluidDynamicsApplication/trilinos_extension/custom_python/add_trilinos_strategies_to_python.cpp

```diff
@@ -47,8 +47,12 @@ void AddTrilinosStrategiesToPython(pybind11::module& m)
     using TrilinosFSStrategy = FSStrategy< TrilinosSparseSpace, UblasLocalSpace, TrilinosLinearSolver>;
     py::class_< TrilinosFSStrategy, typename TrilinosFSStrategy::Pointer, TrilinosBaseSolvingStrategy >(m,"TrilinosFSStrategy")
     .def(py::init< ModelPart&, BaseSolverSettings&, bool >())
+    .def(py::init< ModelPart&, BaseSolverSettings&, bool, bool >())
     .def(py::init< ModelPart&, BaseSolverSettings&, bool, const Kratos::Variable<int>& >())
-    .def("CalculateReactions",&TrilinosFSStrategy::CalculateReactions)
+    .def(py::init< ModelPart&, BaseSolverSettings&, bool, bool, const Kratos::Variable<int>& >())
+    .def("CalculateReactions", [](TrilinosFSStrategy& self) {
+        KRATOS_WARNING("TrilinosFSStrategy") << "\'CalculateReactions()\' exposure is deprecated. Use the constructor with the \'CalculateReactionsFlag\' instead." << std::endl;
```

Comment:
Why depreciating and not completely removing this ?

Reply:
Backwards compatibility.